portal.go

```diff
@@ -2169,13 +2169,25 @@ func (portal *Portal) convertMediaMessage(intent *appservice.IntentAPI, source *
 		//	portal.log.Errorfln("Failed to send media retry receipt for %s: %v", info.ID, err)
 		//}
 		converted.Error = database.MsgErrMediaNotFound
+
+		errorText := "Old photo or attachment."
+		if portal.bridge.Config.Bridge.HistorySync.BackfillMedia {
+			if len(portal.bridge.Config.Bridge.HistorySync.Media) > 0 {
+				errorText += " Media will be requested from your phone later."
+			} else {
+				errorText += ` React with the \u267b (recycle) emoji or the text "click to retry" to request this media from your phone or use the backfill command to request all missing media.`
+			}
+		} else {
+			errorText += " Media backfill is disabled. Ask your server administrator to enable it."
+		}
```

Comment:
That config flag doesn't currently disable media backfill, the reaction is always enabled